src/analyzing/AST_to_IL.ml

```diff
@@ -241,13 +241,13 @@ let name_of_entity ent =
       Some name
   | _____else_____ -> None
 
-let composite_of_container : G.container_operator -> IL.composite_kind =
+let composite_of_container ~e_gen : G.container_operator -> IL.composite_kind =
```

Comment:
~e_gen? not very readable label.

Reply:
I don't like it that much either, but do you have a suggestion that is not super long? I was thinking to rename them to `g_expr` (_generic expr_). In some places we use `eorig` but now there is an actual `orig` type and I'm trying to reserve "orig" for values of that type.